libsrc/plus4/crt0.s

```diff
@@ -65,6 +64,15 @@ L1:     lda     sp,x
         sta     ENABLE_RAM
         stx     $FFFE           ; Install interrupt handler
         sty     $FFFF
+        ldx     IRQVec
+        ldy     IRQVec+1
+        stx     IRQInd+1
+        sty     IRQInd+2
+        ldx     #<IRQStub
+        ldy     #>IRQStub
+        stx     IRQVec
+        sty     IRQVec+1
```

Comment:
The general cc65 style is to use A/X for addresses. A doesn't need to be preserved here anymore.